packages/local_auth/local_auth_android/example/android/app/build.gradle

```diff
@@ -25,7 +25,7 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 33
```

Comment:
The red is on `local_auth` compilation, not `local_auth_android`. That's why it was post-publish (from c9f6bee7177ce532555282c27f4e22ff8017c409); if it were in the `_android` subpackage it would have shown up in presubmit. So we'll need to update `local_auth`'s example.

@camsim99 we'll need to be careful about updating the examples of app-facing packages as well when we do this kind of dependabot roll.

Reply:
Oh, shoot I see. Sorry about missing that. So, in the future, we need to bump in the app facing package as well as the android sub-package?

Reply:
ah, that test isn't running here.

Reply:
> Oh, shoot I see. Sorry about missing that.

I didn't think of it either. Unfortunately we still have holes in presubmit testing of things that only break dependencies, because running *all* of our tests twice would be a lot. But it seems like we still need to find the right line.

> So, in the future, we need to bump in the app facing package as well as the android sub-package?

If we know something will affect any clients of the plugin, then all clients in our repo (which is usually just the app-facing package; occasionally we have cross-plugin dependencies in examples, like `camera`'s example depending on `video_player`) should be updated.

The trick is knowing when it will affect clients, but having to update the in-package example is a good warning sign.

Reply:
That makes sense. Would it be helpful to provide some guidance on this in the wiki? I'm not sure how many folks will run into this, though.

Reply:
It wouldn't hurt, but I suspect we'll hit it infrequently enough that we'll forget even if we write it down.

I left https://github.com/flutter/flutter/issues/52115#issuecomment-1422536521 to capture considering adding CI that would catch this.